src/sentinel1_reader/sentinel1_burst_slc.py

```diff
@@ -218,7 +220,31 @@ def as_isce3_radargrid(self):
                                                  width,
                                                  ref_epoch)
 
-    def to_vrt_file(self, out_path):
+    def slc_to_gtiff(self, out_path):
```

Comment:
this can be general. Does not need to be GeoTiff. Can be any gdal format with default to ENVI.
I suggest to change the name of the function to 
write_to_disk or write_sls or write or even a better name if you or others can think of one? 
then have the output format as an input argument default to ENVI.